ui-v2/app/router.js

```diff
@@ -49,11 +49,8 @@ export const routes = {
         addresses: {
           _options: { path: '/addresses' },
         },
-        tags: {
-          _options: { path: '/tags' },
-        },
-        metadata: {
-          _options: { path: '/metadata' },
+        meta: {
+          _options: { path: '/meta' },
```

Comment:
Oh sorry I don't think I was super clear about this, I think we should keep the `/metadata` URL here (so not `meta`).